modules/atatus/vue/plugins/atatusRum.client.ts

```diff
@@ -0,0 +1,24 @@
+import type { App } from 'vue'
+import * as atatus from 'atatus-spa'
+import ATATUS_RUM_API_KEY from '#ioc/config/ATATUS_RUM_API_KEY'
+export default async (app: App) => {
+  function trackVueExceptions(app: App) {
```

Comment:
Either extract this function out of the default export or remove it and leave the code inlined.